src/components/ShareToken/ShareToken.js

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import { ReactComponent as ShareIcon } from '../../images/icon-share.svg';
+import { ReactComponent as CopyIcon } from '../../images/icon-copy.svg';
+import './ShareToken.css';
+
+const ShareToken = ({ token }) => {
+  const [showMobileShare, setShowMobileShare] = useState(false);
+
+  function copyToken() {
+    navigator.clipboard.writeText(token).catch((err) => {
+      document.execCommand(token); // possible fallback for older browsers
+      console.log(err);
+    });
+  }
```

Comment:
What do we think of adding some feedback for the user when they click the copy button? Should be pretty easy to do with the below logic, followed by an additional `<label>` on the `shareToken` input (I've put this second part in a separate comment at the relevant spot below). 

```suggestion
const ShareToken = ({ token }) => {
  const [showMobileShare, setShowMobileShare] = useState(false);
  const [copySuccess, setCopySuccess] = useState(false);

  function copyToken() {
    navigator.clipboard.writeText(token).catch((err) => {
      document.execCommand(token); // possible fallback for older browsers
      console.log(err);
    });
    setCopySuccess(true);
  }
```

Reply:
Ooh, fantastic idea, I love it!

Reply:
@nick-zanetti Thanks for the reminder earlier today that github uses this kind of feature. I ended up looking at their markup and CSS to see how they did it (was curious about how they handled a11y), so slightly modified approach but inspired by your suggested code. Check it out!